src/components/list/examples/list-action.tsx

```diff
@@ -20,6 +20,14 @@ export class ListActionExample {
             value: 1,
             icon: 'gorilla',
             actions: this.actionItems,
+            component: {
+                name: 'limel-circular-progress',
+                props: {
+                    value: 5,
+                    maxValue: 10,
+                    suffix: '%',
+                },
+            },
```

Comment:
Do we really need this to show up in every example? I think it should be enough to have one specific example where we show how to use a custom component in the list

Reply:
check the last commit message ;)
This was just to make testing easier. We will drop this commit.